docs/en/latest/router-radixtree.md

```diff
@@ -71,7 +71,96 @@ Here are the rules:
 |/blog/foo/gloo | `/blog/foo/*` |
 |/blog/bar | not match |
 
-#### 4. Parameter match
+#### 4. Different routes have the same uri
+
+When different routes have the same uri, you can set the priority attribute of the route to determine which route to match first, or add matching rules to distinguish different routes.
+
+1. Different routes have the same uri and set the `priority` attribute
+
+Create two routes with different `priority` values ​​(the larger the value, the higher the priority).
```

Comment:
We should mention the `priority` field is prior to the "other matching rules"

Reply:
updated.